docs/25_developer.md

```diff
@@ -173,6 +174,22 @@ level, complete with versioning information. Note that we prefer that
 this dependency list is automatically generated and kept up-to-date by
 the build system.*
 
+### Apache Spark
```

Comment:
Wrong Spark. The one we're using is Apache-licensed, but is not an Apache project: http://sparkjava.com/